beets/ui/__init__.py

```diff
@@ -134,24 +134,25 @@ def print_(*strings, **kwargs):
 
     txt = u' '.join(strings)
     txt += kwargs.get('end', u'\n')
+    stream = kwargs.get('stream', sys.stdout)
 
     # Encode the string and write it to stdout.
     if six.PY2:
         # On Python 2, sys.stdout expects bytes.
         out = txt.encode(_out_encoding(), 'replace')
-        sys.stdout.write(out)
+        stream.write(out)
     else:
         # On Python 3, sys.stdout expects text strings and uses the
         # exception-throwing encoding error policy. To avoid throwing
         # errors and use our configurable encoding override, we use the
         # underlying bytes buffer instead.
-        if hasattr(sys.stdout, 'buffer'):
+        if hasattr(stream, 'buffer'):
             out = txt.encode(_out_encoding(), 'replace')
-            sys.stdout.buffer.write(out)
+            stream.buffer.write(out)
         else:
             # In our test harnesses (e.g., DummyOut), sys.stdout.buffer
             # does not exist. We instead just record the text string.
-            sys.stdout.write(txt)
+            stream.write(txt)
```

Comment:
Great. Now that you're using the logging infrastructure, I believe these changes to `print_` are unnecessary. Could you please back them out?